### PR TITLE
BWS-99: + success-messages

### PR DESCRIPTION
**Pull request by ania-leichenko**
Screenshot_8

### CODE REVIEW

File: frontend/src/services/notification/notification.module.scss [hunk 1]
```diff
@@ -13,6 +13,15 @@
   border-top: 5px solid var(--color-red);
 }
 
+.successWrapper > div {
+  display: flex;
+  width: 100%;
+  color: #9a9a9a;
+  font-size: 14px;
+  background: var(--color-boltgun);
+  border-top: 5px solid #077304;
```
Comment (Donymak) [inline frontend/src/services/notification/notification.module.scss]: Add this color to variables

**Review by what1s1ove — COMMENTED** [review]
attach screenshot please 🙏

File: frontend/src/services/notification/notification.module.scss [hunk 2]
```diff
@@ -32,3 +48,31 @@
   height: 70px;
   background: center no-repeat url("../../assets/img/attention-icon.svg");
 }
+
+.check {
+  width: 30px;
+  height: 30px;
+  background: center no-repeat url("../../assets/img/check-icon.svg");
```
Comment (junojei) [inline frontend/src/services/notification/notification.module.scss]:
```suggestion
  background: center no-repeat url("/assets/img/check-icon.svg");
```

File: frontend/src/services/notification/notification.module.scss [hunk 3]
```diff
@@ -32,3 +48,31 @@
   height: 70px;
   background: center no-repeat url("../../assets/img/attention-icon.svg");
```
Comment (junojei) [inline frontend/src/services/notification/notification.module.scss]:
```suggestion
  background: center no-repeat url("/assets/img/attention-icon.svg");
```

File: frontend/src/services/notification/notification.module.scss (hunk 1 above)
Comment (what1s1ove) [reply frontend/src/services/notification/notification.module.scss]: agreed

File: frontend/src/services/notification/notification.module.scss (hunk 2 above)
Comment (what1s1ove) [reply frontend/src/services/notification/notification.module.scss]: try this way

File: frontend/src/services/notification/notification.module.scss [hunk 4]
```diff
@@ -32,3 +48,31 @@
...
+  right: 25px;
+}
+
+.successWrapper div[role="button"] span {
+  color: #ffffff;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+body .successWrapper > div > div:last-child > div[role="button"] > span {
```
Comment (what1s1ove) [inline frontend/src/services/notification/notification.module.scss]: try to use `:global` selector to avoid this

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**